packages/client/src/components/FilteringArea/FilteringSection.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const FilteringSection = ({
+  title,
+  data,
+  iconName,
+  checkboxName,
+  selectedValue,
+  setSelectedValue,
+}) => {
+  const iconLink = `/assets/vectors/${iconName}.svg`;
+
+  const handleShowAll = (e) => {
+    const all = document.getElementsByName(checkboxName);
+    for (let i = 0; i < all.length; i += 1) {
+      if (all[i].type === 'checkbox') all[i].checked = false;
+    }
+    e.target.style.borderColor = '#8c5c02';
```

Comment:
This is perfectly fine, however, 😅 
if you like to make it a bit more advanced, you could apply a class to the button, according to the value of what is selected / not selected, and then have the styling controlled by the classes through the styling sheet 
💡